lib/tmp.js

```diff
@@ -121,6 +121,11 @@ function _parseArguments(options, callback) {
  * @private
  */
 function _generateTmpName(opts) {
+
+  // fail early on missing tmp dir
+  if (!opts.dir && !tmpDir)
+    throw new Error("no tmp dir specified");
```

Comment:
Please use single-quotes in strings.

Reply:
Please start the error with a capital letter.

Reply:
done